cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -155,24 +199,40 @@ int main(int argc, char** argv)
         std::cout << "Usage:" << std::endl;
         std::cout << "./sample_cudapoa [-m] [-h]" << std::endl;
         std::cout << "-m : Generate MSA (if not provided, generates consensus by default)" << std::endl;
+        std::cout << "-l : Perform long-read sample (if not provided, will run window-based sample by default)" << std::endl;
```

Comment:
`will run short-read sample by default`

Reply:
good catch, I kept to skip reading it all this time! changed.